### PR TITLE
add preconditions for control plane migration

### PR DESCRIPTION
**Pull request by christianhuening**
**How to categorize this PR?**

/area control-plane-migration
/kind enhancement

**What this PR does / why we need it**:
Adds a brief precondition section to indicate what needs to be done so control plane migration works.

### CODE REVIEW

**Comment by gardener-robot-ci-1** [comment]
Thank you @christianhuening for your contribution. Before I can start building your PR, a member of the organization must set the required label(s) {'reviewed/ok-to-test'}. Once started, you can check the build status in the PR checks section below.

File: docs/usage/control_plane_migration.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Control Plane Migration
 
+## Preconditions
+
+To be able to use this feature you need to enable the feature gate `SeedChange` in your `garden-apiserver`
+by adding the following command flag: `--feature-flags=SeedChange=true`.
```
Comment (rfranzke) [inline docs/usage/control_plane_migration.md]: Wouldn't it be `--feature-gates`?

File: docs/usage/control_plane_migration.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Control Plane Migration
 
+## Preconditions
+
+To be able to use this feature you need to enable the feature gate `SeedChange` in your `garden-apiserver`
```
Comment (rfranzke) [inline docs/usage/control_plane_migration.md]:
```suggestion
To be able to use this feature you need to enable the feature gate `SeedChange` in your `gardener-apiserver`
```

**Review by rfranzke — COMMENTED** [review]
cc @plkokanov @stoyanr

File: docs/usage/control_plane_migration.md (hunk 1 above)
Comment (christianhuening) [reply docs/usage/control_plane_migration.md]: of course, will correct.

**Review by rfranzke — APPROVED** [review]
/lgtm
/reviewed ok-to-test
/milestone v1.31

**Pull request merged**